docs/01-app/02-guides/self-hosting.mdx

```diff
@@ -226,19 +177,15 @@ module.exports = {
 }
 ```
 
-### Version Skew
+## Version Skew
 
 Next.js will automatically mitigate most instances of [version skew](https://www.industrialempathy.com/posts/version-skew/) and automatically reload the application to retrieve new assets when detected. For example, if there is a mismatch in the `deploymentId`, transitions between pages will perform a hard navigation versus using a prefetched value.
 
 When the application is reloaded, there may be a loss of application state if it's not designed to persist between page navigations. For example, using URL state or local storage would persist state after a page refresh. However, component state like `useState` would be lost in such navigations.
 
-Vercel provides additional [skew protection](https://vercel.com/docs/deployments/skew-protection?utm_source=next-site&utm_medium=docs&utm_campaign=next-website) for Next.js applications to ensure assets and functions from the previous version are still available to older clients, even after the new version is deployed.
-
-You can manually configure the `deploymentId` property in your `next.config.js` file to ensure each request uses either `?dpl` query string or `x-deployment-id` header.
```

Comment:
Hmm, can a user or platform enable this feature outside Vercel? If so, could we document how here? If not, maybe remove this section entirely?

Reply:
Let me follow up on this one – agree we need to have more details here, or make it parts of adapters.